### DISCUSSION TITLE
scipy package error 0.10.1

### DISCUSSION THREAD

**Discussion by sordidloam** [neonbjb/tortoise-tts#410] [General]
When attempting to install the requirements, I'm met with 

`        File "C:\Users\Admin\AppData\Local\Temp\pip-install-cingj6cr\scipy_6659b58e63b74f2ab786d6eae0b28831\build\py3k\scipy\setup.py", line 4, in configuration
          config = Configuration('scipy',parent_package,top_path)
        File "C:\Users\Admin\.conda\envs\env_python3.10\lib\site-packages\numpy\distutils\misc_util.py", line 807, in __init__
          raise ValueError("%r is not a directory" % (package_path,))
      ValueError: 'build\\py3k\\scipy' is not a directory
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.`

**Comment by user76765** [comment]
go into requirements.txt and update scipy==newest one. i think its ==1.7.3. you might also get an issue with numpy
